prow/scripts/cluster-integration/helpers/delete-dns-record.sh

```diff
@@ -1,57 +1,12 @@
 #!/usr/bin/env bash
 
-#Description: Removes DNS record with given subdomain from the DNS Zone.
-#
-#Expected vars:
-# - CLOUDSDK_CORE_PROJECT: name of a GCP project containing the Zone with the record.
-# - CLOUDSDK_DNS_ZONE_NAME: Name of the existing DNS zone in the project (NOT it's DNS name!)
-# - DNS_FULL_NAME: DNS name
-# - IP_ADDRESS: v4 IP Address of the DNS record.
-#
-#Permissions: In order to run this script you need to use a service account with "DNS Administrator" role
-set -o errexit
+set -e
+set -o pipefail
 
-discoverUnsetVar=false
+readonly DEVELOPMENT_DIR="$( cd "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/../../../../development" && pwd )"
 
-for var in CLOUDSDK_CORE_PROJECT CLOUDSDK_DNS_ZONE_NAME DNS_FULL_NAME IP_ADDRESS; do
-    if [ -z "${!var}" ] ; then
-        echo "ERROR: $var is not set"
-        discoverUnsetVar=true
-    fi
-done
+readonly TOOL_DIR=longlastingdnscleaner
+readonly OBJECT_NAME="Long lasting cluster DNS cleaner"
 
-if [ "${discoverUnsetVar}" = true ] ; then
-    exit 1
-fi
-
-attempts=3
-retryTimeInSec="5"
-function deleteDNSWithRetries() {
-    set +e
-
-    for ((i=1; i<=attempts; i++)); do
-        gcloud dns --project="${CLOUDSDK_CORE_PROJECT}" record-sets transaction start --zone="${CLOUDSDK_DNS_ZONE_NAME}" && \
-        gcloud dns record-sets transaction remove "${IP_ADDRESS}" --zone="${CLOUDSDK_DNS_ZONE_NAME}" --name="${DNS_FULL_NAME}" --type=A --ttl=300 && \
-        gcloud dns --project="${CLOUDSDK_CORE_PROJECT}" record-sets transaction execute --zone="${CLOUDSDK_DNS_ZONE_NAME}"
-
-        if [[ $? -eq 0 ]]; then
-            break
-        fi
-
-        gcloud dns record-sets transaction abort --zone="${CLOUDSDK_DNS_ZONE_NAME}" --verbosity none
-
-        if [[ "${i}" -lt "${attempts}" ]]; then
-            echo "Unable to delete DNS record, let's wait ${retryTimeInSec} seconds and retry. Attempts ${i} of ${attempts}."
-        else
-            echo "Unable to delete DNS record after ${attempts} attempts, giving up."
-            exit 1
-        fi
-
-        sleep ${retryTimeInSec}
-    done
-
-    set -e
-}
-
-deleteDNSWithRetries
+"${DEVELOPMENT_DIR}"/resources-cleanup.sh "${TOOL_DIR}" "${OBJECT_NAME}" "$@"
```

Comment:
Please verify which buildpack is used. This script needs Golang in version 1.11.

Reply:
kyma-cluster-infra uses buildpack-golang:v20181204-a6e79be I'd assume based on what we talked about yesterday, this is not a go 1.11 image. Why do you say it needs 1.11?

Reply:
Generally, go 1.11 provided some changes. But if you only execute then it will probably work with the older version.